odoo/models.py

```diff
@@ -4078,7 +4078,7 @@ def create(self, vals_list):
         records = self._create(data_list)
 
         # protect fields being written against recomputation
-        protected = [(data['protected'], data['record']) for data in data_list]
+        protected = [(data['protected'], data['record']) for data in data_list if data['protected']]
```

Comment:
does this actually make a difference?

Reply:
I don't think so (or maybe a tiny difference), it just a shortcut for creates values without any protected fields. 
I am going to remove it.